modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js

```diff
@@ -0,0 +1,260 @@
+                                y: 0
+                            }],
+                            'name': devices[i].name
+                        });
+            }
+        } else {
+            if (isMultilined) {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "x"
+                        },
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "y"
+                        },
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "z"
+                        }
+                );
+            } else {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': $("#connectedcup-details").data("devicename")
+                        });
+            }
+        }
+
+        var graph = new Rickshaw.Graph(graphConfig);
+        graph.render();
+
+        var xAxis = new Rickshaw.Graph.Axis.Time({
+            graph: graph
+        });
+        xAxis.render();
+
+        var yAxis = new Rickshaw.Graph.Axis.Y({
+            graph: graph,
+            orientation: 'left',
+            element: document.getElementById("y_axis-" + type),
+            width: 40,
+            height: 410
+        });
+        yAxis.render();
+
+        var slider = new Rickshaw.Graph.RangeSlider.Preview({
+            graph: graph,
+            element: document.getElementById("slider-" + type)
+        });
+
+        var legend = new Rickshaw.Graph.Legend({
+            graph: graph,
+            element: document.getElementById("legend-" + type)
+        });
+
+        var hoverDetail = new Rickshaw.Graph.HoverDetail({
+            graph: graph,
+            formatter: function (series, x, y) {
+                var date = '<span class="date">' +
+                           moment((x + tzOffset) * 1000).format('Do MMM YYYY h:mm:ss a') + '</span>';
+                var swatch = '<span class="detail_swatch" style="background-color: ' +
```

Comment:
Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings